docs/settings.rst

```diff
@@ -143,3 +144,11 @@ Default behavior of the Google Cloud PubSub library is to use 10 threads per sub
 We thought this was too much for a default setting and have taken the liberty of
 reducing the thread count to 2. If you would like to maintain the default Google PubSub
 library behavior, please set this value to 10.
+
+``FILTER_SUBS_BY``
+----------------------------
+
+**Optional**
+
+Boolean function that applies a global filter on all subscriptions.
```

Comment:
Here we could explain that it can be overridden by defining a filter in the sub decorator